packages/components/src/Tooltip/tests/chromatic/Tooltip.stories.tsx

```diff
@@ -0,0 +1,242 @@
+
+const buttonText = "Hover me";
+const childrenText = "More info";
+
+const meta = {
+    title: "Components/Tooltip",
+    component: Tooltip,
+    args: {
+        children: childrenText
+    },
+    decorators: [
+        (Story, context) => {
+            if (context.parameters.skipGlobalDecorator) {
+                return <Story />;
+            }
+
+            return (
+                <Flex UNSAFE_marginBottom="4rem" UNSAFE_marginTop="3rem" justifyContent="center">
+                    <Story />
+                </Flex>
+            );
+        }
+    ]
+} satisfies Meta<typeof Tooltip>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default = {
+    render: args => (
+        <TooltipTrigger defaultOpen>
+            <Button>{buttonText}</Button>
+            <Tooltip {...args} />
+        </TooltipTrigger>
+    )
+} satisfies Story;
+
+export const Placement = {
+    render: args => (
+        <Grid gap="core_480"
+            justifyItems="center"
+            templateColumns={["1fr", "1fr"]}
+            width="100%"
+        >
+            <TooltipTrigger defaultOpen placement="start">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+            <TooltipTrigger defaultOpen placement="end">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+            <TooltipTrigger defaultOpen placement="right">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+            <TooltipTrigger defaultOpen placement="left">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+            <TooltipTrigger defaultOpen placement="top">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+            <TooltipTrigger defaultOpen placement="bottom">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+        </Grid>
+    )
+} satisfies Story;
+
+export const ShouldFlip = {
+    render: args => (
+        <Stack>
+            <H1>Original Placement: left</H1>
+            <TooltipTrigger defaultOpen placement="left">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+        </Stack>
+    ),
+    decorators: [
+        Story => (
+            <Flex justifyContent="left">
+                <Story />
+            </Flex>
+        )
+    ],
+    parameters: {
+        skipGlobalDecorator: true
+    }
+} satisfies Story;
+
+export const LinkTrigger = {
+    render: args => (
+        <TooltipTrigger defaultOpen>
+            <Link>{buttonText}</Link>
+            <Tooltip {...args} />
+        </TooltipTrigger>
+    )
+} satisfies Story;
+
+export const AvatarTrigger = {
```

Comment:
You should replace this example with IconTrigger where you use a SVG. It's a better usecase to test instead of Avatar. In our avatar group, we will want the avatar to be rendered inside a button, so it can be used on mobile + accessible via keyboard